stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>The last sentence</p><p> </p><blockquote>By testing all our models on the test set that is the latter half of the season </blockquote><p> </p><p>is the first time the test set being the latter half of the season is mentioned. I think it should be mentioned earlier, which could  simply be done by adding something like</p><p> </p><blockquote>We choose an unusually large fraction, <strong>the last</strong> 50% of our data, to be the test set</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Done</p>